spec.html

```diff
@@ -24888,16 +24890,16 @@ <h1>Runtime Semantics: BindingClassDeclarationEvaluation ( ): either a normal co
       <emu-grammar>ClassDeclaration : `class` BindingIdentifier ClassTail</emu-grammar>
       <emu-alg>
         1. Let _className_ be StringValue of |BindingIdentifier|.
-        1. Let _value_ be ? ClassDefinitionEvaluation of |ClassTail| with arguments _className_ and _className_.
-        1. Set _value_.[[SourceText]] to the source text matched by |ClassDeclaration|.
+        1. Let _sourceText_ be the source text matched by |ClassDeclaration|.
+        1. Let _value_ be ? ClassDefinitionEvaluation of |ClassTail| with arguments _className_, _className_, and _sourceText_.
         1. Let _env_ be the running execution context's LexicalEnvironment.
         1. Perform ? InitializeBoundName(_className_, _value_, _env_).
         1. Return _value_.
       </emu-alg>
       <emu-grammar>ClassDeclaration : `class` ClassTail</emu-grammar>
       <emu-alg>
-        1. Let _value_ be ? ClassDefinitionEvaluation of |ClassTail| with arguments *undefined* and *"default"*.
-        1. Set _value_.[[SourceText]] to the source text matched by |ClassDeclaration|.
+        1. Let _sourceText_ be the source text matched by |ClassDeclaration|.
+        1. Let _value_ be ? ClassDefinitionEvaluation of |ClassTail| with arguments *undefined*, *"default"*, and _sourceText_.
```

Comment:
We can return directly in these cases instead of introducing an alias.

```suggestion
        1. Return ? ClassDefinitionEvaluation of |ClassTail| with arguments *undefined*, *"default"*, and _sourceText_.
```